src/components/subscription/SubscriptionItem.tsx

```diff
@@ -0,0 +1,82 @@
+import { SUBSCRIPTION_ITEM_STYLE } from '@/constants/styles';
+import type { Platform } from '@/types/recording';
+import { MinusIcon } from 'lucide-react';
+import { useState } from 'react';
+import ConfirmModal from '../common/ConfirmModal';
+import AlertModal from '../common/AlertModal';
+import useUnsubscribeMutation from '@/hooks/useUnsubscribeMutation';
+
+type SubscriptionItemProps = {
+  subscriptionId: number;
+  subscribedAt: string | null;
+  channel: {
+    channelId: string;
+    channelName: string;
+    platform: Platform;
+  };
+};
+
+const SubscriptionItem = (props: SubscriptionItemProps) => {
+  const { subscriptionId, subscribedAt, channel } = props;
+  const [alertMessage, setAlertMessage] = useState('');
+  const [showConfirmModal, setShowConfirmModal] = useState(false);
+
+  const confirmMessage = `[${channel.channelName}] 채널 구독을 해지하시겠습니까?`;
+
+  const { mutate: mutateUnsubscribe } = useUnsubscribeMutation(
+    () => setAlertMessage('채널 구독이 해지되었습니다.'),
+    () => setAlertMessage('구독채널 해지 도중 문제가 발생했습니다.')
+  );
+
+  const handleClickButton = () => {
+    setShowConfirmModal(true);
+  };
+
+  const handleConfirm = () => {
+    setShowConfirmModal(false);
+    mutateUnsubscribe(subscriptionId);
+  };
+
+  return (
+    <>
+      <div className={SUBSCRIPTION_ITEM_STYLE.container}>
+        <div className={SUBSCRIPTION_ITEM_STYLE.box}>
+          <div className={SUBSCRIPTION_ITEM_STYLE.leftBox}>
+            <p
+              className={SUBSCRIPTION_ITEM_STYLE.channel}
+            >{`채널: ${channel.channelName}`}</p>
+            <p
+              className={SUBSCRIPTION_ITEM_STYLE.platform}
+            >{`플랫폼: ${channel.platform}`}</p>
+          </div>
+          <div className={SUBSCRIPTION_ITEM_STYLE.rightBox}>
+            <span className={SUBSCRIPTION_ITEM_STYLE.dateText}>
+              {subscribedAt}
+            </span>
+            <button
+              onClick={() => handleClickButton()}
+              className={SUBSCRIPTION_ITEM_STYLE.deleteButton}
+            >
+              <MinusIcon className={SUBSCRIPTION_ITEM_STYLE.deleteIcon} />
+            </button>
+          </div>
+        </div>
+      </div>
+      {showConfirmModal && (
+        <ConfirmModal
+          message={confirmMessage}
+          onConfirm={handleConfirm}
+          onCancel={() => setShowConfirmModal(false)}
```

Comment:
```suggestion
          onCancel={handleCloseModal}
```
```javascript
const handleCloseModal = () => setAlertMessage('');
<AlertModal message={alertMessage} onClose={handleCloseModal} />
```
밑에 [src/pages/Home/Subscription.tsx] 에서는 이렇게 한 줄이더라도 handler 함수로 빼셨더라구요!

그런데 또 여기서는 handler 함수로 안 빼고 계신 것 같아서요!

통일성이 있어야 하지 않을까요???

Reply:
```ts
          onCancel={() => setShowConfirmModal(false)}
```
함수 로직이 `setShowConfirmModal(false)`뿐이라서 별도 handle함수로 분리할 필요가 없을 것 같습니다.
[src/pages/Home/Subscription.tsx]에서도 동일하게 적용하도록 하겠습니다.🙂